tests/rkt_privatenet_test.go

```diff
@@ -522,5 +632,34 @@ func TestPrivateNetCustomMacvlan(t *testing.T) {
 			Subnet: "10.1.2.0/24",
 		},
 	}
-	testPrivateNetCustomDual(t, net)
+	testPrivateNetCustomDual(t, nt)
+}
+
+func TestPrivateNetCustomBridge(t *testing.T) {
+	ifaces, err := net.Interfaces()
+	if err != nil {
+		t.Fatalf("Cannot get network host's interfaces: %v", err)
+	}
+
+	var ifaceName string
+	if len(ifaces) < 2 {
+		t.Skipf("Cannot run test without non-lo host interface")
+	}
+	ifaceName = ifaces[1].Name
```

Comment:
It's not the best thing to assume that ifaces[0] will be loopback and just take the second iface. Let's at least take the first interface that's not a loopback and is up. `net.Interfaces()` returns `net.Interface` which has `Flags()` method. It can be used to determine if the interface is loopback and up.